images/scripts/fedora.setup

```diff
@@ -114,7 +122,18 @@ maybe firewall-cmd --permanent --add-port 8765/tcp
 
 echo 'NETWORKING=yes' > /etc/sysconfig/network
 
-useradd -c Administrator -G wheel admin
+# These build from a cloud image, set up network-scripts for eth1 to avoid long boot hang on
+# NetworkManager-wait-online.service
+if [ "$IMAGE" = "fedora-34" ]; then
+    printf 'BOOTPROTO=none\nDEVICE=eth1\nONBOOT=no\n' > /etc/sysconfig/network-scripts/ifcfg-eth1
+    # our cloud-init.iso does not set up the host name
+    echo "127.0.1.1 $(hostname)" >> /etc/hosts
+fi
+
+# admin user exists in the cloud image
+if [ "$IMAGE" != fedora-34 ]; then
```

Comment:
Right, we specifically told it so through our cloud-init.iso. Could you please use the same approach as in rhel.setup?
```sh
# cloud images already create the user through cloud-init.iso
if ! getent passwd admin >/dev/null; then
    useradd -c Administrator -G wheel admin
    echo foobar | passwd --stdin admin
fi
```